sdk/iOS/src/MSCoreDataStore.m

```diff
@@ -214,7 +207,6 @@ -(MSSyncContextReadResult *)readWithQuery:(MSQuery *)query orError:(NSError *__a
         fr.entity = entity;
         fr.predicate = query.predicate;
         fr.sortDescriptors = query.orderBy;
-        fr.resultType = NSDictionaryResultType;
```

Comment:
Do you know the perf difference in flipping from this to the new model?  Are they roughly eqv?